pkg/operator/configobservation/images/observe_images.go

```diff
@@ -148,6 +148,46 @@ func ObserveAllowedRegistriesForImport(genericListers configobserver.Listers, re
 	return observedConfig, errs
 }
 
+func ObserveImagestreamImportMode(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
```

Comment:
is it possible to use `lister := configobserver.ImageConfigLister` directly?

Reply:
the ImageConfigLister is under the Listers object [here](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/ae690fe574be44fae506bfcc59caa629fce9a803/pkg/operator/configobservation/interfaces.go#L15) which is passed through [here](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/ae690fe574be44fae506bfcc59caa629fce9a803/pkg/operator/configobservation/configobservercontroller/observe_config_controller.go#L39). I believe that is why it is done this way?

Reply:
@tkashem is my observation correct, or is there another way to do it?